components/RegistrationScreen/index.jsx

```diff
@@ -62,11 +62,11 @@ class RegistrationScreen extends Component {
     this.setState({ update: true })
   }
 
-  onBusinessPhoneChange({ country, externalChange }) {
+  onBusinessPhoneChange({ country, externalChange, isValid }) {
     const { vm } = this.props
     const { country: previousSelectedCountry } = this.state
 
-    if (!country || !country.code) {
+    if (!isValid) {
```

Comment:
Let's keep the requirement for `country` and `country.code`. So this would be additional requirement to the regexp.

Reply:
@yoution is it possible to keep the requirement to enter country code here?

Reply:
you mean change code to ` if(!country || !country.code || !isValid)` ?
this is the another reason we can not use validate rule, in validate rule, we can not access the country.code